thunder/dynamo/report.py

```diff
@@ -464,3 +471,194 @@ def helper_backend(gm, example_inputs):
     compiled(*args, **kwargs)
 
     return FXReport(graphs)
+
+
+class ThunderSegmentGraphReport(FXGraphReport):
+    def __init__(self, graph, graph_name, compiled_fn, example_input, thunder_options: dict, split_reason: str):
+        super().__init__(graph, graph_name)
+        self.compiled_fn = compiled_fn
+        self.example_input = example_input
+        self.thunder_options = thunder_options
+        self.split_reason = split_reason
+
+        self.fusion_reports: list[ThunderFusionReport] = []
+        self.fwd_trc = None
+        self.bwd_trc = None
+
+    def create_thunder_traces(self):
```

Comment:
Looks like this is supposed to be a private method. In that case, it would be nice to prefix with `_`.